src/tutorial/helmholtz.jl

```diff
@@ -107,37 +90,42 @@ vp, vecp = eigen(Matrix(A), Matrix(B))
-@assert all(results .≈ ref_results) "Invalid Helmholtz results" #hide
-
-# Now we can export the solution (the eigenvectors) at nodes of the mesh for several eigenvalues.
-# We will restrict to the first 20 eigenvectors. To do so, we will create a `FEFunction` for each
-# eigenvector. This `FEFunction` can then be evaluated on the mesh centers, nodes, etc.
-ϕ = FEFunction(U)
-nvecs = min(20, get_ndofs(U))
-values = zeros(nnodes(mesh), nvecs)
-for ivec in 1:nvecs
-    set_dof_values!(ϕ, vecp[:, ivec])
-    values[:, ivec] = var_on_vertices(ϕ, mesh)
-end
-
-# To write a VTK file, we need to build a dictionnary linking the variable name with its
-# values and type
-using WriteVTK
-out_dir = joinpath(@__DIR__, "../../myout/helmholtz") # output directory
-mkpath(out_dir) #hide
-dict_vars = Dict("u_$i" => (values[:, i], VTKPointData()) for i in 1:nvecs)
-write_vtk(joinpath(out_dir, "rectangle_mesh"), 0, 0.0, mesh, dict_vars)
+
+# Now we can export the solution (the eigenvectors) at nodes of the mesh for several eigenvalues.   #md
+# We will restrict to the first 20 eigenvectors. To do so, we will create a `FEFunction` for each   #md
+# eigenvector. This `FEFunction` can then be evaluated on the mesh centers, nodes, etc.             #md
+ϕ = FEFunction(U)                               #md
+nvecs = min(20, get_ndofs(U))                   #md
+values = zeros(nnodes(mesh), nvecs)             #md
+for ivec in 1:nvecs                             #md
+    set_dof_values!(ϕ, vecp[:, ivec])           #md
+    values[:, ivec] = var_on_vertices(ϕ, mesh)  #md
+end                                             #md
+
+# To write a VTK file, we need to build a dictionnary linking the variable name with its #md
+# values and type #md
+using WriteVTK #md
+outputvtk = "../../myout/helmholtz/rectangle_mesh"                          #md
+mkpath(dirname(joinpath(@__DIR__, outputvtk)))                              #src
+dict_vars = Dict("u_$i" => (values[:, i], VTKPointData()) for i in 1:nvecs) #md
+write_vtk(joinpath(@__DIR__, outputvtk), 0, 0.0, mesh, dict_vars)           #md
 
 # And here is the eigenvector corresponding to the 4th eigenvalue:
 # ![](../assets/helmholtz_x21_y21_vp6.png)
 
+if get(ENV, "TestMode", "false") == "true"                  #src
+    results = sqrt.(abs.(vp[3:8]))                          #src
+    ref_results = [                                         #src
+        3.144823462554393,                                  #src
+        4.447451992013584,                                  #src
+        6.309054755690625,                                  #src
+        6.309054755690786,                                  #src
+        7.049403274103087,                                  #src
+        7.049403274103147,                                  #src
+    ]                                                       #src
+    @test all(results .≈ ref_results)                       #src
+    files2check = outputvtk .* [".pvd", "_00000000.vtu"]    #src
```

Comment:
if it's to show how we could compare the results using binary files ok. But in this precise case (helmholtz), I think the comparison with the ref results is enough (and better). In my opinion, checking the file hash should be avoid when possible : if `WriteVTK` decide to add one comment in the file, all these tests will fail.

Reply:
I agree and feel free to correct it in a future PR. And yes, for now I am only focusing on setting up the whole process (with file checking and so on)

Reply:
Done !